jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -66,6 +66,11 @@
      */
     private final ReceiverConstraintsMap receiverVideoConstraintsMap = new ReceiverConstraintsMap();
 
+    /**
+     * The map of source name -> receiver endpoint id -> video constraints.
```

Comment:
I have expanded the `ReceiverConstraintsMap` into the `receiver endpoint id -> video constraints` part, but removed it now since it appears to be confusing.